docs/components/radio.md

```diff
@@ -60,17 +64,76 @@ Will place label text to the right of the radio. Can also be [slotted](#slots).
 
 ### description
 
-Will place description text under the radio label (required). Can also be [slotted](#slots).
+Will place description text under the radio label. Can also be [slotted](#slots).
 
 ```html
 <KRadio :selected-value="true" v-model="radio" label="Label Example" description="Some subheader text" />
 ```
 
 <KRadio :selected-value="true" v-model="radio" label="Label Example" description="Some subheader text" />
 
-### selectedValue
+### type
 
-The value of the `KRadio` option that will be emitted by the `change` and `update:modelValue` events.
+Controls appearance of radio input element. Accepted values:
+- `card`
```

Comment:
I see your point, totally valid concern but I thought this approach might leave some room for adding more types of card in the future - maybe a color selector or different style of regular radio input.

![Screen Shot 2023-01-25 at 9 12 03 AM](https://user-images.githubusercontent.com/36751160/214586252-4f5cf7a1-b390-4aaa-b40e-abf772f6a537.png)
![Screen Shot 2023-01-25 at 9 12 29 AM](https://user-images.githubusercontent.com/36751160/214586258-80d7c4b8-cfd9-4159-a35d-c2fecb10ffc6.png)

Reply:
Hmm yeah I guess I agree - pushed changes.
Fyi @kaiarrowood - I liked your idea of keeping `''` for default, but I agree with Adam that it's better to have more descriptive values in case we'll have more types in future.